sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/BooleanLiteralUnnecessary.Fixed.cs

```diff
@@ -37,6 +37,12 @@ public BooleanLiteralUnnecessary(bool a, bool b, bool? c)
             x = a;                 // Fixed
             x = !a;                  // Fixed
             x = false;             // Fixed
+            x = false;             // Fixed
+            x = Foo();             // Fixed
+            x = Foo();             // Fixed
+            x = Foo();              // Fixed
```

Comment:
[cosmetic] Could you align the `// Fixed` comments. This will means that the `// Noncompliant` are also aligned.

Reply:
I tried for 5 minutes and failed and won't do it :( sorry.

Reply:
Aaaah!